mungegithub/submit-queue/pv.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  labels:
+    app: submit-queue
+  name: submit-queue-github-cache
+spec:
+  capacity:
+    storage: 1Gi
+  accessModes:
+    - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Recycle
```

Comment:
!!! this should definitely be "Retain", so we don't erase the cache on disconnect